src/components/TextField/TextField.jsx

```diff
@@ -105,6 +109,7 @@ const TextField = createClass({
 			debounceLevel: 500,
 			lazyLevel: 1000,
 			value: '',
+			autoComplete: 'off',
```

Comment:
Seems to me like we should probably enable it by default and only default disable it for cases like drop menu and other selects. I just wonder what impact this might have on users who do actually care about the default browser's autocomplete for text fields.

Reply:
Ok sure makes sense to me.  `SearchableMultiSelect`, `SearchableSelect`, `SearchableSingleSelect` -- any others I'm missing?   Can you think of any situations where these components would want autocomplete?  I'm trying to decide if we want it to be drivable by a prop.